draft-irtf-cfrg-vdaf.md

```diff
@@ -4326,19 +4319,27 @@ state across evaluations. See {{idpf-bbcggi21}} for details.
 ### Encoding inputs as indices {#poplar1-idpf-index-encoding}
 
 How data are represented as IDPF indices is up to the application. When the
-inputs are fixed-length byte strings, the most natural choice of encoder is
-`from_be_bytes()`. This ensures that, when a string is a prefix of another, so
-too is its index. (Index prefixes are defined in {{idpf}}). For example,
+inputs are fixed-length byte strings, the most natural choice of representation
+is as a bit string formed from all the bits of the byte string, first ordered by
+byte position, then ordered from most significant bit to least significant bit
+within each byte. This ensures that, when a byte string is a prefix of another,
+so too is its corresponding index. (Index prefixes are defined in {{idpf}}). For
```

Comment:
This parenthesis is never closed
```suggestion
so too is its corresponding index. Index prefixes are defined in {{idpf}}). For
```

Reply:
It is closed, though it's clustered with curly braces here. This is carried over from existing text.

Reply:
Oh, I think I missed it because the period comes after.

Normally if I have a paranthetical that is an entire sentence, I put the period with in the paranthetical.